src/tests/lib/components/TooltipTest.svelte

```diff
@@ -1,7 +1,9 @@
 <script>
   import Tooltip from "$lib/components/Tooltip.svelte";
+
+  export let text = "text";
```

Comment:
Shouldn't this be the same type as in the tooltip to allow for `undefined`?

Reply:
Done